mu-plugins/10up-plugin/.npmrc

```diff
@@ -0,0 +1,2 @@
+engine-strict=true
+package-lock=false
```

Comment:
So I'll tell you a non-obvious behavior. 

If this isn't added, the `package-lock` can be created at the package level rather than the root level. From my testing, this happens on npm 7 but not on npm 8 which, no matter where you install from, always goes to the root.

Now, if you go and install without any package-lock from a package with this setup, the root gets a package-lock.json instead of the package getting one. I'd rather not advertise that it's OK to install dependencies from packages with these safeguards and stick to the idea that ONLY installing from the root is the way.

Can't foresee any downside to this change. Is there anything I'm missing? Would you rather wait till NPM 8 is normalized?